tests/integration-tests/configs/schedulers.yaml

```diff
@@ -160,6 +160,12 @@ test-suites:
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
           oss: ["alinux2"]
           schedulers: [ "slurm" ]
+    test_slurm.py::test_slurm_custom_config_parameters:
+      dimensions:
+        - regions: ["euw1-az1"]
+          instances: ["c5.xlarge"]
```

Comment:
Not really sure why.
We do not need any special instance type for this test.

I believe we should use ODCR if and only if we strictly need. So that, if there is no real need we can release it.
Otherwise we will create a false need and keep it even if there are no real requirements.